pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -1701,6 +1701,48 @@ func TestWaitOnPermit(t *testing.T) {
 	}
 }
 
+func TestListPlugins(t *testing.T) {
+	tests := []struct {
+		name    string
+		plugins *config.Plugins
+		// If noPlugin is true, we expect 'ListPlugins' method return empty plugin except queue sort plugin and bind plugin.
+		noPlugin bool
```

Comment:
what about pluginsCount

Reply:
Do you mean adding the 'pluginsCount'  as a test parameter?

Reply:
I have added `pluginsCount` and remove `noPlugin`.